addons/sale_timesheet/models/project_project.py

```diff
@@ -331,6 +331,43 @@ def _get_sale_order_items_query(self, domain_per_model=None):
         ]))
         return query
 
+    def _get_items_id_per_section_id(self):
+        return {
+            'materials': [],
+            'billable_fixed': [],
+            'billable_milestones': [],
+            'billable_time': [],
+            'billable_manual': [],
+        }
+
+    def _get_domain_from_section_id(self, section_id):
+        section_domains = {
+            'materials': [
+                ('product_id.type', '!=', 'service')
+            ],
+            'billable_fixed': [
+                ('product_id.type', '=', 'service'),
+                ('product_id.invoice_policy', '=', 'order')
+            ],
+            'billable_milestones': [
+                ('product_id.type', '=', 'service'),
+                ('product_id.invoice_policy', '=', 'delivery'),
+                ('product_id.service_type', '=', 'milestones'),
+            ],
+            'billable_time': [
+                ('product_id.type', '=', 'service'),
+                ('product_id.invoice_policy', '=', 'delivery'),
+                ('product_id.service_type', '=', 'timesheet'),
+            ],
+            'billable_manual': [
+                ('product_id.type', '=', 'service'),
+                ('product_id.invoice_policy', '=', 'delivery'),
+                ('product_id.service_type', '=', 'manual'),
+            ],
+        }
+
+        return section_domains.get(section_id, [])
```

Comment:
So you don't need to call `_get_sale_items_domain()` anymore now?

Reply:
nope, its still needed. good catch.